Header header
float64 est_z
float64 kinect_z
float64 out_z
bool using_est